angular.json

```diff
@@ -26,12 +26,7 @@
             "main": "src/main.ts",
             "tsConfig": "src/tsconfig.app.json",
             "polyfills": "src/polyfills.ts",
-            "assets": [
```

Comment:
@gilclefes It's hard to see which lines was modified in this file...can you just rework on this and commit only lines changed

Reply:
for this the this was the only change  **"sourceMap": true** the rest are automatic formating